src/System.Devices.I2c/II2cDevice.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Devices.I2c
+{
+    // TODO: Document
+    public interface II2cDevice : IDisposable
+    {
+        void Write(byte[] buffer);
```

Comment:
Should these take `Span<byte>`?

Reply:
I believe so.  Honestly, I'm still trying to get a better understanding on Span.  I'm hoping working on this project helps me in this area.  I changed the byte[] to Span<byte> for now as I wasn't sure if there are some that could be ReadOnlySpan<byte>.